src/Apps/Collect2/Routes/Collection/Components/CollectionsHubRails/FeaturedCollectionsRails/index.tsx

```diff
@@ -23,6 +23,29 @@ interface Props {
   collectionGroup: FeaturedCollectionsRails_collectionGroup
 }
 
+export const renderCarousel = (
```

Comment:
I pulled out the `renderCarousel` func to be able to test that it gets called with the correct value.